layouts/partials/script.html

```diff
@@ -1,12 +1,6 @@
-<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-MML-AM_SVG"></script>
-<script type="text/x-mathjax-config">
-    MathJax.Hub.Config({
-            showMathMenu: false, //disables context menu
-            tex2jax: {
-            inlineMath: [ ['$','$'], ['\\(','\\)'] ]
-           }
-    });
-</script>
+{{ if or .Params.math .Site.Params.math }}
```

Comment:
just add this param to config.yml as example to users. 
Parameter names need to have semantics.like this:
```yml
params:
  enableMathJax: true
```

Reply:
Site wide param is named `enableMathJax`, for a single post they can set `math: true`, you might wanna add info about this in README.